WordPress/src/main/java/org/wordpress/android/datasets/ReaderDatabase.java

```diff
@@ -53,6 +53,7 @@
      *   99 - added feed_url to tbl_blog_info
      *  100 - changed primary key on tbl_blog_info
      *  101 - dropped is_reblogged from ReaderPostTable
+     *  102 - changed primary key of tbl_blog_info from blog_id+feed_id to just blog_id
```

Comment:
> What is the reason behind this change?

Sorry, I should've explained that above. The old calls would return a `blog_id` of zero for all feeds, and since these had a non-zero `feed_id` it was necessary to make the primary key include both those columns. The new calls *do* return a `blog_id` for feeds, so the primary key only needs to be on `blog_id`.